README.md

```diff
@@ -61,6 +67,9 @@ These dependencies are compatible with Java Standard Edition (SE).
 </dependency>
 ```
 
+![](gifs/pom.gif "Pom gif")
```

Comment:
I think .gif for adding dependencies in a pom are not necessary, and does not especially bring value compared to the documentation